fission/src/index.css

```
@@ -45,12 +45,20 @@ a:hover {
     color: #c26c37;
 }
 
+html {
+    overflow: hidden;
+    position: relative;
+    height: 100%;
+}
 body {
```

Tiny nitpick but there should be a blank line here.

```suggestion
}

body {
```